pinot-servicemanager/Dockerfile

```diff
@@ -1,6 +1,5 @@
 # Choose libraries we need from Pinot's image.
-# TODO: 0.5.0-rc2 does not have parallel start: 21a372b2f58f9c6e27fe9710d6952ed519342061
-FROM apachepinot/pinot:0.5.0-SNAPSHOT-4c20bb567-20200902 as install
+FROM apachepinot/pinot:0.5.0-SNAPSHOT-e0ed179c2-20200910 as install
```

Comment:
note: 0.5.0 isn't in docker yet

Reply:
pfff 21a372b2f58f9c6e27fe9710d6952ed519342061 is not in release-0.5.0 :(

Reply:
This is good news, that means there are still improvements coming!